ott/geometry/pointcloud.py

```diff
@@ -608,8 +617,7 @@ def to_LRCGeometry(
       Otherwise, returns the re-scaled low-rank geometry.
     """
     if self.is_squared_euclidean:
-      (n, m), d = self.shape, self.x.shape[1]
-      if n * m > (n + m) * d:  # here apply_cost using LRCGeometry preferable.
+      if self.check_LRC_dim:
```

Comment:
Test are failing ATM because it's now private.

Reply:
yes, thanks! sync'ed and left to do something :)